force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.js

```diff
@@ -357,6 +357,7 @@ export default class NewServiceSchedule extends LightningElement {
                     initialStartDate.setMonth(today.getMonth());
                     initialStartDate.setDate(today.getDate());
                     initialStartDate.setFullYear(today.getFullYear());
+                    this.handleMonthlyRecurrenceOptions(initialStartDate);
```

Comment:
Do we want to do this for all recurrence types though? Won't this run on weekly and everything else?